web-app/django/VIM/apps/main/forms.py

```diff
@@ -0,0 +1,52 @@
+from django import forms
+from django.contrib.auth.forms import (
+    AuthenticationForm,
+    UserCreationForm,
+    PasswordChangeForm,
+)
+from django.contrib.auth.models import User
+
+
+class LoginForm(AuthenticationForm):
+    username = forms.EmailField(label="Email Address")
+    password = forms.CharField(widget=forms.PasswordInput, label="Password")
+
+    class Meta:
+        model = User
+        fields = ("username", "password")
+
+
+class RegistrationForm(UserCreationForm):

```

Comment:
I think you are just using the default behaviour here, right? So I don't think you need to add the configurations?